WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1573,7 +1572,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         return cell;
     }
 
-    if (section.category == BlogDetailsSectionCategoryMigrationSuccess && MigrationSuccessCardView.shouldShowMigrationSuccessCard == YES) {
+    if (section.category == BlogDetailsSectionCategoryMigrationSuccess) {
```

Comment:
Thanks for the detailed explanation!

Yup, that's why we want a representation of a view declared separately and used as a single source of truth, instead of having separate logic if statements inside one of the `tableView` delegate methods. This `shouldShowMigrationSuccessCard == YES` was out of place.